general/getting-started-in-tech.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: Getting Started in Tech
+---
+
+Technology is a vast subject. It can enable you to write a to-do list on your phone, sell goods online or send a space probe to Neptune. Software is built by developers and codebar is here to teach you how to write such software. Whether you want to make a career of it or use your skills on personal projects.
+
+Typically software can fall into two (very broad) categories:
+
+- **User interface** - How to present information to people using the technology and enable them to interact with it. For example:
+    - Displaying your vehicle and responding to joystick inputs in a game of Mario Kart
+    - Displaying your balance and allowing you to make a payment in your bank’s online banking site
+    - Displaying current temperature and allowing you to change the target on your Hive or Nest smart thermostat
+- **Data processing** - How to manipulate the data and perform the task that the software implements. For example:
+    - Calculating the position of your vehicle and detecting collisions
+    - Validating that your account has enough funds to send this payment and processing the payment
+    - Starting or stopping your heating or air conditioning depending on the current temperature and the target you set
+
+At codebar, we concentrate on web technologies for several reasons:
+
+- It doesn't need any initial investment: all the tools you need to learn about web technologies available online for free. And the computer you use doesn’t have to be new or a top of the range model, most computers will perform well, including shared computers in libraries.
```

Comment:
Small update, just because we refer to multiple web technologies but then use singular.
```suggestion
- No initial investment is needed: all the tools you need to learn about web technologies available online for free. And the computer you use doesn’t have to be new or a top of the range model; most computers will perform well, including shared computers in libraries.
```